### DISCUSSION TITLE
何时发布v1.0，可以应用到生产环境？

### DISCUSSION THREAD

**Discussion by cnLiuMing** [Tencent/tdesign-vue-next#1557] [Ideas]
何时发布v1.0，可以应用到生产环境？

**Comment by PengYYYYY** [comment]
目前我们正在完善我们的单元测试，待单元测试补充完善后会发布正式版本，可以来参与https://github.com/Tencent/tdesign-vue-next/issues/768

**Comment by liect** [comment]
我已经应用到生产环境了,是有些小问题,但影响不大,有问题随时提lssues

**Comment by PengYYYYY** [comment]
已发布1.0版本，1.0版本会进行长期维护。破坏性变更迭代将放在2.0版本。

**Reply by liect** [reply]
好突然啊
